metrics/report/mail.go

```diff
@@ -0,0 +1,87 @@
+/// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"net/smtp"
+	"strings"
+)
+
+// bodyMsg concatenates the output of
+// checktool in one string, this string
+// is the body of the email message.
+func bodyMsg(outRows []string) string {
+	var body string
+
+	body += "<h3>PnP Checktool results</h3>\n"
```

Comment:
I'd like us to consider if we need html emails here - would plain text not serve equally well at this point?
The report summary currently has no rich text embedded (although it would be nice if it could highlight the failure points more clearly) - and, unless we are currently encoding the summary table into a fixed width font (which I can't see in the code, but maybe I missed it or it is a subtlety), might this mess up the fixed width table formatting?